data/binding/pref_helper.go

```diff
@@ -70,3 +79,26 @@ func (m *preferencesMap) preferencesChanged(p fyne.Preferences) {
 		item.checkForChange()
 	}
 }
+
+func (m *preferencesMap) migratePreferences(p1, p2 fyne.Preferences) {
```

Comment:
This look like it is moving data from fyne.Preferences to another. Maybe using src/dst instead of p1/p2 would make it more readable.